gateway-contracts/contracts/CiphertextCommits.sol

```diff
@@ -154,10 +156,9 @@ contract CiphertextCommits is
             // A ciphertext handle should only be added once, ever
             $._isCiphertextMaterialAdded[ctHandle] = true;
 
-            // As explained above, a "late" valid coprocessor could still see its transaction sender
-            // address be added to the list after consensus. This var is here to make the
-            // `getCiphertextMaterials` and `getSneCiphertextMaterials` methods retrieve this list
-            // more easily
+            // A "late" valid coprocessor could still see its transaction sender address be added to
+            // the list after consensus. This variable is here to be able to retrieve this list later
+            // by only knowing the handle, since a consensus can only happen once per handle
             $._ctHandleConsensusHash[ctHandle] = addCiphertextHash;
 
             emit AddCiphertextMaterial(
```

Comment:
Now that we have the `getAddCiphertextMaterialConsensusTxSenders`, shouldn't we remove the list of `coprocessorTxSenderAddresses` from the event to reduce gas? Maybe also the ciphertext digests? 🤔